.github/workflows/platformDeploy.yml

```diff
@@ -12,6 +12,10 @@ env:
   SHOULD_DEPLOY_PRODUCTION: ${{ github.event_name == 'release' }}
   DEVELOPER_DIR: /Applications/Xcode_14.1.app/Contents/Developer
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name }}
+  cancel-in-progress: true
+
```

Comment:
What is this change for?

Reply:
Whoops, that's not supposed to be there...